CartesianPoint.java

```diff
@@ -1,21 +1,21 @@
 /**
  * Initialized the class for Cartesian points on a plane.
  */
-public class cPoint {
+public class CartesianPoint {
 
     private double x;
     private double y;
     private double radius;
```

Comment:
as far as I can tell, radius is used only in printPolar. You are recomputing it every time you construct / move, even if you only print it once. Also it incurs extra space on the object